examples/centralised-logging/kubeflow/seldon-analytics-kubeflow.yaml

```diff
@@ -10,4 +10,6 @@ grafana:
       - kubeflow-gateway.kubeflow.svc.cluster.local
   extraEnv:
     #replace with KF gateway URI
-    GF_SERVER_ROOT_URL: "%(protocol)s://%(domain)s/grafana"
\ No newline at end of file
+    GF_SERVER_ROOT_URL: "%(protocol)s://%(domain)s/grafana"
```

Comment:
This is already in master too https://github.com/SeldonIO/seldon-core/blob/master/examples/centralised-logging/kubeflow/seldon-analytics-kubeflow.yaml#L13